kong/plugins/galileo/buffer.lua

```diff
@@ -254,7 +256,9 @@ function _M:add_entry(...)
 
   if err >= self.queue_size then -- err is the queue size in this case
      ok, err = self:flush()
-     if not ok then return nil, err end -- for our tests only
+     if not ok then
+       return nil, err
+     end -- for our tests only
```

Comment:
We should probably move this comment inside the branch, as per tacit convention across the codebase.